functions/src/index.ts

```diff
@@ -1,3 +1,17 @@
-import * as http_api from './api/http/api';
+import * as httpApi from "./api/http/api";
+import * as authOnCreate from "./api/auth/on-create";
+import {ServiceAccount, cert, initializeApp} from "firebase-admin/app";
+import * as dotenv from "dotenv";
 
-exports.http_api = http_api.default;
\ No newline at end of file
+// load environment variables
+dotenv.config({path: "../.env"});
+const serviceAccount = JSON.parse(process.env.FIREBASE_SERVICE_ACCOUNT || "{}");
+
+// initialize firebase app
```

Comment:
Can't we use the firebase/server.ts here? If needed, we can add changes to read the env in different ways